gatling-http-client/src/main/java/io/gatling/http/client/body/form/FormUrlEncodedRequestBody.java

```diff
@@ -61,7 +61,11 @@ private StringBuilder encode() {
     for (Param param : content) {
       encodeAndAppendFormParam(sb, param.getName(), param.getValue(), charset);
     }
-    sb.setLength(sb.length() - 1);
+    if (sb.length() > 0) {
+      sb.setLength(sb.length() - 1);
+    } else {
+      sb.setLength(sb.length());
+    }
```

Comment:
```suggestion
    sb.setLength(Math.max(0, sb.length() - 1))
```

or

```suggestion
    if (sb.length() > 0) {
      sb.setLength(sb.length() - 1);
    }
```

no need to set to the current value, does it?

Reply:
Oh, it makes sense. I checked and found that `StringBuilder` from the pool has a length - 0